terraform/aio/vm.tf

```diff
@@ -16,10 +16,15 @@ variable "aio_vm_image" {
   default = "CentOS-stream8"
 }
 
-variable "aio_vm_keypair" {
+variable "aio_vm_user" {
```

Comment:
Unused

Reply:
Good spot, thanks. Just to mention I'm not 100% sure that removing configure-local-networking will work, but thought I'd try it again now that the CI was passing. Will revert if it fails.